modules/templates/proc-creating-templates.adoc

```diff
@@ -1,15 +1,13 @@
 // Module included in the following assemblies:
 //
-// * admin/assembly-admin-templates.adoc
+// * assemblies/assembly-admin-templates.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="proc-creating-templates_{context}"]
 = Creating a template by using the Template Editor
 
 You can create a template by using the Template Editor.
```

Comment:
[Future] This statement is redundant to the heading. It would be nice to add some conceptual information about templates and why a user would want to create one.

Reply:
There's some conceptual info in the assembly: https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-219/admin-rhdh/#assembly-admin-templates

This bit is really just to differentiate between different methods of creating templates, since you can do that a couple of ways :)